qds_sdk/cloud/aws_cloud.py

```diff
@@ -0,0 +1,168 @@
+        self.compute_config['compute_access_key'] = compute_access_key
+        self.compute_config['compute_secret_key'] = compute_secret_key
+        self.compute_config['role_instance_profile'] = role_instance_profile
+
+    def set_location(self,
+                     aws_region=None,
+                     aws_availability_zone=None):
+        self.location['aws_region'] = aws_region
+        self.location['aws_availability_zone'] = aws_availability_zone
+
+    def set_network_config(self,
+                           bastion_node_public_dns=None,
+                           persistent_security_groups=None,
+                           master_elastic_ip=None,
+                           vpc_id=None,
+                           subnet_id=None):
+        self.network_config['bastion_node_public_dns'] = bastion_node_public_dns
+        self.network_config['persistent_security_groups'] = persistent_security_groups
+        self.network_config['master_elastic_ip'] = master_elastic_ip
+        self.network_config['vpc_id'] = vpc_id
+        self.network_config['subnet_id'] = subnet_id
+
+    def set_cloud_config_from_arguments(self, arguments):
+        self.set_cloud_config(compute_access_key=arguments.compute_access_key,
+                              compute_secret_key=arguments.compute_secret_key,
+                              use_account_compute_creds=arguments.use_account_compute_creds,
+                              aws_region=arguments.aws_region,
+                              aws_availability_zone=arguments.aws_availability_zone,
+                              role_instance_profile=arguments.role_instance_profile,
+                              vpc_id=arguments.vpc_id,
+                              subnet_id=arguments.subnet_id,
+                              persistent_security_groups=arguments.persistent_security_groups,
+                              bastion_node_public_dns=arguments.bastion_node_public_dns,
+                              master_elastic_ip=arguments.master_elastic_ip)
+
+    def create_parser(self, argparser):
+
+        # compute settings parser
+        compute_config = argparser.add_argument_group("compute config settings")
+        compute_creds = compute_config.add_mutually_exclusive_group()
+        compute_creds.add_argument("--enable-account-compute-creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_true",
+                                   default=None,
+                                   help="to use account compute credentials")
+        compute_creds.add_argument("--disable-account-compute-creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_false",
+                                   default=None,
+                                   help="to disable account compute credentials")
+
```

Comment:
nit: Remove an extra line here. Also please take care of this thing elsewhere in the PR.